digiwf-integrations/digiwf-okewo-integration/digiwf-okewo-integration-starter/src/main/java/de/muenchen/oss/digiwf/okewo/integration/configuration/OkEwoIntegrationAutoConfiguration.java

```diff
@@ -1,93 +1,109 @@
 package de.muenchen.oss.digiwf.okewo.integration.configuration;
 
-import de.muenchen.oss.digiwf.okewo.integration.gen.ApiClient;
+import de.muenchen.oss.digiwf.okewo.integration.client.ApiClient;
+import de.muenchen.oss.digiwf.okewo.integration.client.api.PersonApi;
+import de.muenchen.oss.digiwf.okewo.integration.client.api.PersonErweitertApi;
 import de.muenchen.oss.digiwf.okewo.integration.properties.OkEwoIntegrationProperties;
-import de.muenchen.oss.digiwf.okewo.integration.service.PropertiesServiceTemplate;
-import de.muenchen.oss.digiwf.okewo.integration.gen.api.PersonApi;
-import de.muenchen.oss.digiwf.okewo.integration.gen.api.PersonErweitertApi;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.FilterType;
-import org.springframework.web.client.RestTemplate;
+import org.springframework.web.reactive.function.client.ExchangeFilterFunctions;
+import org.springframework.web.reactive.function.client.WebClient;
 
+import java.net.MalformedURLException;
+import java.net.URL;
 
+@Slf4j
 @Configuration
 @RequiredArgsConstructor
 @ComponentScan(
-        basePackages = "de.muenchen.oss.digiwf.okewo.integration",
-        excludeFilters = {
-                @ComponentScan.Filter(
-                        type = FilterType.ASSIGNABLE_TYPE,
-                        classes = {
-                                /**
-                                 * Exclude to avoid multiple instantiation of beans with same name.
-                                 * This class is instantiated in {@link OkEwoIntegrationAutoConfiguration}
-                                 * to give the bean another name.
-                                 */
-                                ApiClient.class,
-                                PersonApi.class,
-                                PersonErweitertApi.class
-                        }
-                )
-        }
+    basePackages = "de.muenchen.oss.digiwf.okewo.integration",
+    excludeFilters = {
+        @ComponentScan.Filter(
+            type = FilterType.ASSIGNABLE_TYPE,
+            classes = {
+                /**
+                 * Exclude to avoid multiple instantiation of beans with same name.
+                 * This class is instantiated in {@link OkEwoIntegrationAutoConfiguration}
+                 * to give the bean another name.
+                 */
+                ApiClient.class,
```

Comment:
this is strange and should not be needed. you may component scan the same component multiple time. why is this needed?